vnext/Microsoft.ReactNative/IReactNotificationService.idl

```diff
@@ -8,96 +8,87 @@ import "IReactPropertyBag.idl";
 
 namespace Microsoft.ReactNative {
 
-  // A subscription to a notification.
-  // The subscription is removed when this object is deleted or the Unsubscribe method is called.
   [webhosthidden]
-  DOC_STRING("A subscription to a notification. The subscription is removed when this object is deleted or the @.Unsubscribe method is called.")
+  DOC_STRING(
+    "A subscription to a notification.\n"
+    "The subscription is removed when this object is deleted or the @.Unsubscribe method is called.")
   interface IReactNotificationSubscription
   {
-    // Name of the notification.
     DOC_STRING("Name of the notification.")
     IReactPropertyName NotificationName { get; };
 
-    // The IReactDispatcher provided when the notification subscription created.
-    // All notifications will be handled using this dispatcher.
-    DOC_STRING("The @IReactDispatcher provided when the notification subscription created. All notifications will be handled using this dispatcher.")
+    DOC_STRING(
+      "The @IReactDispatcher provided when the notification subscription created.\n"
+      "All notifications will be handled using this dispatcher.\n"
+      "In case if it is null, the events are handled synchronously.")
     IReactDispatcher Dispatcher { get; };
 
-    // True if the subscription is still active.
-    // This property is checked before notification handler is invoked.
-    DOC_STRING("True if the subscription is still active. This property is checked before notification handler is invoked.")
+    DOC_STRING(
+      "True if the subscription is still active.\n"
+      "This property is checked before notification handler is invoked.")
     Boolean IsSubscribed { get; };
 
-    // Remove the subscription.
-    // Because of the multi-threaded nature of the notifications, the handler can be still called
-    // after the Unsubscribe method called if the IsSubscribed property is already checked.
-    // Consider calling the Unsubscribe method and the handler in the same IReactDispatcher
-    // to ensure that no handler is invoked after the Unsubscribe method call.
-    DOC_STRING("Remove the subscription. Because of the multi-threaded nature of the notifications, the handler can be still called \
-after the Unsubscribe method called if the @.IsSubscribed property is already checked. \
-Consider calling the Unsubscribe method and the handler in the same @IReactDispatcher \
-to ensure that no handler is invoked after the Unsubscribe method call.")
+    DOC_STRING(
+      "Removes the subscription.\n"
+      "Because of the multi-threaded nature of the notifications, the handler can be still called "
+      "after the @.Unsubscribe method called if the @.IsSubscribed property is already checked. "
+      "Consider calling the @.Unsubscribe method and the handler in the same @IReactDispatcher "
+      "to ensure that no handler is invoked after the @.Unsubscribe method call.")
```

Comment:
Unclear what the "IsSubscribed property is already checked" means. Is it this? Also, by handler do you mean the instance of ReactNotificationHandler passed in to Subscribe?

```suggestion
      "Because of the multi-threaded nature of notifications, the handler can be still called "
      "after the @.Unsubscribe method has been called (e.g. if the @.IsSubscribed property has already been checked). "
      "Consider calling the @.Unsubscribe method and the @ReactNotificationHandler on the same @IReactDispatcher "
      "to ensure that no handler is invoked after the @.Unsubscribe method call.")
```